### DISCUSSION TITLE
After the update, the color palette has changed

### DISCUSSION THREAD

**Discussion by optic0n** [neovim/neovim#29087] [Q&A]
![Screenshot_2024-05-28_10-13-50](https://github.com/neovim/neovim/assets/77533678/a64e106e-350a-47f4-a825-d6f75ccf5e37)
After updating neovim to version 10, my colors changed, as you can see in the photo. On the left, I have arch linux with version NVIM v0.10.0 and I have ubuntu with version NVIM v0.7.2 on the right. For some reason, the color of the autocomplete menu changed, and also some white line was added to the nerdtree plugin. Now I set an iceberg color scheme, but regardless of the color scheme, I have these broken colors.
Any thoughts?
I hope you understand what my problem is.

**Comment by clason** [comment]
Yes. That is a feature. See `:h news`.

**Reply by optic0n** [reply]
Is it possible to somehow remove all this, to return the past appearance? Because I don't like the new colors.

**Reply by clason** [reply]
Did you read the changelog as I told you? Because that explains to you exactly what to do. (`colorscheme vim`, if you -- for whatever reason -- prefer the Vim default colors)

**Reply by optic0n** [reply]
Yes, I read it. But if I set `colorscheme vim`, it will change my color scheme. I want to keep the iceberg color scheme. I want to return everything to the way it was before, you can see how it was on the screenshot on the right. Is it possible to return the colors that were before version 0.10? I hope you understand me.

**Reply by clason** [reply]
Set colorscheme vim before setting iceberg; possibly iceberg must be updated to do that after their `hi clear`. Otherwise your only option is to stick to Nvim 0.9.5. (The problem is that Iceberg doesn't set a full colorscheme but implicitly relies on some default links, which it shouldn't because it's not robust.)

**Reply by optic0n** [reply]
Opened an issue in the iceberg repository, thanks for the help.

**Comment by dundargoc** [comment]
I think you should open an issue in the [iceberg](https://github.com/cocopon/iceberg.vim) repository and mention this. They should easily be able to fix this IIUC.

**Reply by optic0n** [reply]
Thanks, opened an issue.
